build.sbt

```diff
@@ -41,13 +40,13 @@ lazy val common = crossProject.
   .jvmSettings(
     libraryDependencies ++= Seq(
       // "com.typesafe.akka" %% "akka-actor" % "2.5.6"
-      "com.typesafe.akka" %% "akka-actor" % "2.5.12"
+      "com.typesafe.akka" %% "akka-actor" % "2.5.13"
       // "com.typesafe.akka" %% "akka-actor" % "2.5-SNAPSHOT"
     )
   )
   .jsSettings(
     libraryDependencies ++= Seq(
-      "org.akka-js" %%% "akkajsactor" % "1.2.5.12-PERF"
+      "org.akka-js" %%% "akkajsactor" % "1.2.5.12"
```

Comment:
this should become `1.2.5.13` as well :-)

Reply:
Lol，I forgot it:).